raiden/tests/unit/test_upgrade.py

```diff
@@ -90,22 +102,30 @@ def test_sequential_version_numbers(tmp_path):
         storage.update_version()
         storage.conn.close()
 
-    with ExitStack() as stack:
-        stack.enter_context(patch(
-            'raiden.utils.upgrades.UPGRADES_LIST',
-            new=upgrade_functions,
-        ))
-        stack.enter_context(patch(
-            'raiden.constants.RAIDEN_DB_VERSION',
-            new=19,
-        ))
```

Comment:
Looks like this was not working properly: https://circleci.com/gh/raiden-network/raiden/33404?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link